pkg/pillar/types/domainmgrtypes.go

```diff
@@ -103,6 +107,9 @@ type DomainStatus struct {
 	LastErrTime        time.Time
 	BootFailed         bool
 	AdaptersFailed     bool
+	IsContainer        bool   // Is this Domain for a Container?
+	ContainerImageID   string // SHA-512 of rkt container image
+	PodUUID            string // Pod UUID outputted by rkt
```

Comment:
Isn't this a single number per EVE device?

Reply:
No @nordmark -- I think you're still confusing kubernetes pods and rkt pods

Reply:
Currently, 

1 Domain = 1 App Instance = 1 POD = 1 Container ( For containers ) = 1 Container Image

Ideally, we will evolve to:

1 App Instance = 1 User Container
1 Domain = 1 Pod ( This object needs to be introduced )

Each Domain / Pod will have multiple App Instances ( Containers )

I believe this ( or some version of it ) is the next step in container evolution?? 
Till then, we can stick the POD UUID as an attribute of Domain??